google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -762,6 +778,13 @@ public StorageClass getStorageClass() {
     return labels;
   }
 
+  /**
+   * Returns the default Cloud KMS key to be applied to newly inserted objects in this bucket.
```

Comment:
Is there an online doc the user can look up for details about what the KMS keys do?

Reply:
Not yet.

Reply:
[Documentation](https://cloud.google.com/storage/docs/encryption/using-customer-managed-keys) is now available.